packages/jellyfish-transaction/src/script/dftx/dftx_pool.ts

```diff
@@ -41,13 +38,15 @@ export class CPoolSwap extends ComposableBuffer<PoolSwap> {
       ComposableBuffer.single<Script>(() => ps.toScript, v => ps.toScript = v, v => new CScript(v)),
       ComposableBuffer.varUInt(() => ps.toTokenId, v => ps.toTokenId = v),
       {
-        fromBuffer (buffer: SmartBuffer) {
+        fromBuffer: (buffer: SmartBuffer): void => {
           const integer = readBigNumberUInt64(buffer)
           const fraction = readBigNumberUInt64(buffer)
-          ps.maxPrice = { integer, fraction }
+          ps.maxPrice = integer.plus(fraction.dividedBy(ONE_HUNDRED_MILLION))
```

Comment:
Something wrong here.

1. I think the `integer` part do not have to to divide by 1e8, it is not in dust aka satoshi.
2. `fraction` number should not affect `integer` (I believe, @fuxingloh @monstrobishi please help to confirm). Current implementation, if the `fraction` bytes execeed 1e8, it will change the combined result.
3. if (2) is true, most likely there should be a checker if `fraction` > 1e8, cap the resulted maxPrice at xxxxxxx.99999999

Reply:
1. In that case, only the fraction is divided by 1e8
2 & 3. That's right, I will add a check to limit fraction to 1e8

Reply:
The checker should be here instead of in `toBuffer`. The calculated `fraction.div(1e8)` should be less than unity. ie the raw buffer should not exceed 0x5F5E0FF.

Reply:
It makes sense to check before writing to buffer instead of when you read it back. 
If you take a look at `writeBigNumberUInt64` and `readBigNumberUInt64`, this is the current implementation.
I could check on both side but I am not sure that it is needed.

Reply:
Fair enough, I will check on both sides